src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -115,6 +117,28 @@ def datetime_type(string):
     date_format = '%Y-%m-%dT%H:%MZ'
     return datetime.strptime(string, date_format)
 
+def json_string_type(raw):
+    """ Loads a JSON string as an object and converts all keys to snake case """
+
+    regex1 = re.compile('(.)([A-Z][a-z]+)')
+    regex2 = re.compile('([a-z0-9])([A-Z])')
+
+    def _to_snake(s):
+        s1 = regex1.sub(r'\1_\2', s)
+        return regex2.sub(r'\1_\2', s1).lower()
+
+    def _convert_to_snake_case(item):
```

Comment:
I went ahead and moved to_camel_case to an external method in _util so other places can use it. I moved this method into there and renamed it 'get_json_object'.